### PR TITLE
method added to parse json directly from string value

### PR DESCRIPTION
**Pull request by s4kibs4mi**

### CODE REVIEW

File: src/main/kotlin/com/beust/klaxon/Parser.kt [hunk 1]
```diff
@@ -90,6 +90,11 @@ class Parser {
         }
     }
 
+    fun parse(rawValue: StringBuilder): Any? {
+        val inputStream: InputStream = ByteArrayInputStream(rawValue.toString().toByteArray())
+        return parse(inputStream)
+    }
```
Comment (cbeust) [inline src/main/kotlin/com/beust/klaxon/Parser.kt]: I think a `use` would safer here:

```
    fun parse(rawValue: StringBuilder): Any? =
        ByteArrayInputStream(rawValue.toString().toByteArray()).use {
             parse(it)
         }
```

Comment (s4kibs4mi) [inline src/main/kotlin/com/beust/klaxon/Parser.kt]: modified please check :)

**Comment by s4kibs4mi** [comment]
thanks :)

**Comment by cbeust** [comment]
Thank you!

**Pull request merged**